src/python/nimbusml/pipeline.py

```diff
@@ -2265,6 +2266,39 @@ def load_model(self, src):
         self.model = src
         self.steps = []
 
+    def __getstate__(self):
+        odict = {'export_version': 1}
+
```

Comment:
Nice! Pls add versioning in base_pipeline_item.py too

Reply:
I have added versioning to `base_pipeline_item` in commit https://github.com/pieths/NimbusML/commit/a34ec151405f756fc9d14e2af768a858241d6ba4